### ISSUE TITLE
"image-feature-extraction" "Xenova/clip-vit-base-patch32" no longer works on v3.2.2

### ISSUE THREAD

**Issue by xitanggg** [huggingface/transformers.js#1112]
### System Info

transformers.js: v3.2.2
OS: Windows
env: browser + Node

### Environment/Platform

- [X] Website/web-app
- [ ] Browser extension
- [X] Server-side (e.g., Node.js, Deno, Bun)
- [ ] Desktop app (e.g., Electron)
- [ ] Other (e.g., VSCode extension)

### Description

👋Thanks for the great work on this project to make it easy to run transformer models on web and Node.js💯

I notice `image-feature-extraction` for the `Xenova/clip-vit-base-patch32` model no longer works correctly after updating transformers.js version from `v3.1.2` to `v3.2.2` and throws the error `Missing the following inputs: input_ids, attention_mask.` if you run the following example code. I can reproduce this issue in both web and Node.js
```typescript
const imageFeatureExtractor = await pipeline(
	'image-feature-extraction',
	'Xenova/clip-vit-base-patch32'
);
const features = await imageFeatureExtractor(url);
```

### Reproduction

1. Install the latest version of transformers.js `npm i @huggingface/transformers`
2. Run the following example code
```typescript
const imageFeatureExtractor = await pipeline(
	'image-feature-extraction',
	'Xenova/clip-vit-base-patch32'
);
const features = await imageFeatureExtractor(url);
```
3. Observe the error `Missing the following inputs: input_ids, attention_mask.`

**Comment by emojiiii** [comment]
import { pipeline } from '@huggingface/transformers';

const classifier = await pipeline('zero-shot-image-classification', 'Xenova/clip-vit-base-patch32');
const url = 'https://huggingface.co/datasets/Xenova/transformers.js-docs/resolve/main/tiger.jpg';
const output = await classifier(url, ['tiger', 'horse', 'dog']);
// [
//   { score: 0.9993917942047119, label: 'tiger' },
//   { score: 0.0003519294841680676, label: 'horse' },
//   { score: 0.0002562698791734874, label: 'dog' }
// ]

[reference](https://huggingface.co/Xenova/clip-vit-base-patch32)

**Comment by xenova** [comment]
Thanks for the report @xitanggg! This was indeed a regression and https://github.com/huggingface/transformers.js/pull/1114 will fix it. I'll add a set of unit tests for this too!
